backend/app/controllers/books_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BooksController < ApplicationController
+  def new
+    @book = Book.new
+  end
+
+  def create
+    @book = Book.new(book_params)
+
+    if @book.save
+      render json: @book, status: :created, location: @book
+    else
+      render json: @book.errors, status: :unprocessable_entity
+    end
+  end
+
+  private
+    def book_params
+      params.require(:book).permit(:title, :description).tap do |book_params|
```

Comment:
```suggestion
      params.require(:book).permit(:title, :description)
```

It's fine to just `permit`. It'll be your model's `validation` responsibility to say if certain parameters are missing when created.